compiler/testdata/pass1.p4

```diff
@@ -0,0 +1,57 @@
+/*
+ Copyright 2023 Google LLC
```

Comment:
Update to 2024? And for the other P4 files similarly.

Reply:
DONE